### DISCUSSION TITLE
Exposing QuickJS

### DISCUSSION THREAD

**Discussion by soulzidda** [cashapp/zipline#1477] [General]
Good afternoon, 

i'm just enquiring in regards to your plans for still exposing quick JS. Is this still something your planning on leaving in place for the foreseeable future or are their plans to stop exposing it soon?

**Comment by JakeWharton** [comment]
QuickJS is fully exposed, but with an opt-in annotation. See https://github.com/cashapp/zipline/blob/b32e6ec85735341e3a4b0522631b93edcd11fe29/zipline/src/hostMain/kotlin/app/cash/zipline/EngineApi.kt#L19-L23

This is because we plan to (someday) switch the engine to Wasm: https://github.com/cashapp/zipline/issues/717

**Reply by soulzidda** [reply]
Thank you for the quick Answer Jake. That is enough for me :).
